tests/conftest.py

```diff
@@ -213,11 +214,48 @@ def kubernetes_client(mocker):
                     "metadata": {
                         "name": "dummy-pod-name",
                         "annotations": {
+                            "hub.jupyter.org/servername": "14399552b42bb5ff",
```

Comment:
They did not exist before and are not used by UI, so, I did not include them.

Reply:
right, but you added a new label `renku.io/username` - is it not needed to get the correct pods in the test?

Reply:
OK, I see, sorry for mis-understanding! 
Actually no, because in tests I am mocking K8s and there is no actual K8s. So, I mock what is returned by the filter result and no actual filter is done, so, there is no need for labels to do a filtering. But, I'll add it just to be consistent with what an actual K8s label looks like.

Reply:
right of course. I think having it here for completeness would still be nice.